src/components/cardViewFlatInfo/cardViewFlatInfo.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Change filename to cardViewFlatInfo to CardViewFlatInfo. Component file name must start with capital letter.